schema/relationship-record.json

```diff
@@ -2,21 +2,21 @@
   "version": "0.4",
-  "title": "Relationship record",
-  "description": "A relationship record is made up of an entity, an interested party (a reference to an entity, natural person, arrangement or trust), details of the interest and provenance information for the statement.",
+  "title": "Relationship record details",
+  "description": "Information about the interests that an interested party (a person or entity) holds in the subject (an entity).",
   "type": "object",
   "properties": {
     "isComponent": {
       "title": "Is component",
-      "description": "Does this Ownership-or-control Statement represent a component of a wider indirect ownership-or-control relationship? Where `isComponent` is 'true': (1) the `statementID` of this secondary Ownership-or-control Statement MUST be an element in the `componentStatementIDs` array of that primary Ownership-or-control Statement, (2) this Ownership-or-control Statement MUST come before that primary Ownership-or-control Statement in a BODS package or stream, (3) the replacement of this Ownership-or-control Statement SHOULD be considered when replacing the primary Ownership-or-control Statement, and (4) the primary Ownership-or-control Statement MUST have a `isComponent` value of 'false'. Where `isComponent` is 'false', this Ownership-or-control Statement is the primary declaration of the relationship between the `subject` and the `interestedParty`.",
+      "description": "Whether this relationship is a component of a wider indirect relationship. Where `isComponent` is 'true': (1) the `recordId` of this secondary Relationship statement MUST be an element in the `componentRecords` array of that primary Relationship statement, (2) this Relationship statement MUST come before that primary Relationship statement in a BODS package or stream, (3) the replacement of this Relationship statement SHOULD be considered when replacing the primary Relationship statement, and (4) the primary Relationship statement MUST have an `isComponent` value of 'false'. Where `isComponent` is 'false', this Relationship statement is the primary declaration of the relationship between the `subject` and the `interestedParty`.",
       "type": "boolean"
     },
-    "componentStatementIDs": {
-      "title": "Component statement IDs",
-      "description": "The identifiers of all component statements that provide detail about the indirect relationship between this Statement's `subject` and `interestedParty`. If this Ownership-or-control Statement has components, it MUST itself have a `isComponent` value of 'false'.",
+    "componentRecords": {
+      "title": "Component record IDs",
+      "description": "The `recordId` values of all component records that provide detail about this indirect relationship between the `subject` and the `interestedParty`. If this relationship has components, its own `isComponent` value MUST be 'false'.",
       "type": "array",
       "items": {
-        "$ref": "urn:components#/$defs/ID"
```

Comment:
@rhiaro - This array is now going to be an array of strings (i.e. recordId values) hence this edit I've made. So I've replaced the line:

```
"$ref": "urn:components#/$defs/ID"
```
But for my interest: can you explain to me how that reference was being resolved and passing the schema tests - since I cant see an 'ID' object defined in components.json?!  Is it resolving to the 'id' property of the 'identifier' object?

Reply:
With the updates to the jsonschema library, it's not possible to simply "check references resolve" in the tests. The way we test this is by making sure there is test data against which to run every part of the schema. This was not picked up as we haven't finished adding in test data yet!

Reply:
Got it! Thanks, @rhiaro.